libldrparser.py

```diff
@@ -204,35 +214,42 @@ def findFile(self, partPath):
             os.path.join(self.libraryLocation, "p", partPath)
         ]
 
-        for path in paths:
-            if os.path.isfile(path):
-                locatedFile = path
-                break
+        # Try the current directory.
+        if os.path.isfile(partPath):
+            locatedFile = partPath
+
+        # Now lets check the list of paths we built earlier.
+        if not locatedFile:
+            for path in paths:
+                if os.path.isfile(path):
+                    locatedFile = path
+                    break
 
         # Failing that, recursively search through every directory
         # in the library folder for the file.
         if not locatedFile:
-            for file in locate(os.path.basename(partPath),
```

Comment:
Technically, `file` is a reserved keyword in certain circumstances. That's why I changed it.